src/components/20-molecules/datepicker/index.js

```diff
@@ -498,8 +497,12 @@ class AXADatepicker extends NoShadowDOM {
       startDate,
     } = this;
 
-    const year = typeof setyear === 'number' ? setyear : _year;
-    const month = typeof setmonth === 'number' ? setmonth : _month;
+    const year =
+      // eslint-disable-next-line no-nested-ternary
```

Comment:
This rule is a good one for my opinion. So, do not disable it.

Reply:
uff